### DISCUSSION TITLE
grand exchange plugin should keep track of history

### DISCUSSION THREAD

**Discussion by an0n716** [runelite/runelite#16126] [Q&A] [answered]
since the ingame history is quite short, its generally not possible to see whether you made or lost money on an item you've held on to for a few days.
I think it would be a useful & fitting feature to have a "history" tab in the plugin side panel that shows every single item you've bought & sold ever, as well as perhaps a search bar to make it easier to look through it.

**Comment by Enriath** [comment, marked as answer]
This is already possible by signing in, and then viewing the history on https://runelite.net/account/grand-exchange

**Reply by an0n716** [reply]
i think i'd be useful to have this info in the client directly 
also without signing in
